client/src/components/Connections/dataWrappers/EntitySearchWrapper.js

```diff
@@ -17,6 +18,7 @@ const EntitySearchWrapper = (WrappedComponent: ComponentType<*>) => {
         {...props}
         entitySearch1={query.eid1 || ''}
         entitySearch2={query.eid2 || ''}
+        showGraph={query.graph || false}
```

Comment:
When you add `graph=false` to the query, this is truthy. Maybe comper it with `"true"`.